core/includes/form.inc

```diff
@@ -4726,8 +4726,14 @@ function theme_textarea($variables) {
 
   // Add resizable behavior.
   if (!empty($element['#resizable'])) {
-    backdrop_add_library('system', 'backdrop.textarea');
-    $wrapper_attributes['class'][] = 'resizable';
+    $valid_values = array('none', 'both', 'horizontal', 'vertical');
+    // Transform previously used boolean value to new string value.
+    if ($element['#resizable'] === TRUE) {
+      $element['#resizable'] = 'vertical';
+    }
+    if (in_array($element['#resizable'], $valid_values)) {
+      $element['#attributes']['class'][] = 'resize-' . $element['#resizable'];
```

Comment:
Should we also keep the `'resizable'` class used before? I don't see any harm in that.

Reply:
Sure, why not, whatever it may be good for... :wink: